woudc_data_registry/models.py

```diff
@@ -904,6 +904,94 @@ def __repr__(self):
         return 'DataRecord({}, {})'.format(self.data_record_id, self.url)
 
 
+class Contribution():
+    """
+    Data Registry Contribution
+
+    """
+    __tablename__ = 'contributions'
+
+    id_field = 'contribution_id'
+    id_dependencies = ['project_id', 'dataset_id', 'station_id',
+                       'instrument_name']
+
+    project_id = Column(String, ForeignKey('projects.project_id'),
+                        nullable=False)
+    contribution_id = Column(String, primary_key=True)
+    dataset_id = Column(String, ForeignKey('datasets.dataset_id'),
+                        nullable=False)
+    station_id = Column(String, ForeignKey('stations.station_id'),
+                        nullable=False)
+    station_name = Column(String, nullable=False)
+    country_id = Column(String, nullable=False)
+    country_name_en = Column(String, nullable=False)
+    country_name_fr = Column(String, nullable=False)
+    instrument_id = Column(String, nullable=False)
+    instrument_name = Column(String,
+                             ForeignKey('instruments.name'), nullable=False)
+    instrument_model = Column(String, nullable=False)
+    start_date = Column(Date, nullable=False)
+    end_date = Column(Date, nullable=False)
+    x = Column(Float, nullable=False)
+    y = Column(Float, nullable=False)
+    z = Column(Float, nullable=False)
+
+    def __init__(self):
+
+        self.project_id = dict()
+        self.contribution_id = ""
```

Comment:
Please use `{}` to create a dictionary (equivalent to calling `dict()`) and `[]` for a list (equivalent to `list()`).

Also, the rest of the project uses single quotes (`'`) for strings rather than double quotes, it's nice to be consistent.